### PR TITLE
Bump dcos-ui package.

### PR DESCRIPTION
**Pull request by rcorral**
****

# Issues

[DCOS-10834](https://mesosphere.atlassian.net/browse/DCOS-10834)
[DCOS-12045](https://mesosphere.atlassian.net/browse/DCOS-12045)

 - [x] Change Log from last:
Logs bug fix, new filtering and funnel for declined offers. **See below for new feature screenshots**
[Diff between commits](https://github.com/dcos/dcos-ui/compare/ffec1877fbfd15e2f44c045d60059107782c6dd2...bbb9d2740bd2067752b33fb383ce60f8f7b6745c)
 - [x] Test Results: https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-ui-pulls/6021/
```
Test Suites: 235 passed, 235 total
Tests:       2790 passed, 2790 total
Snapshots:   0 total
Time:        400.007s
Ran all test suites.
```

Adds Filtering to the services UI page
gif

Adds funnel to declined offers view
img

### CODE REVIEW

**Comment by spahl** [comment]
@rcorral the screenshots are nice but don't remove the template. Keep both

**Comment by spahl** [comment]
Does this have the fixes for logging as well?

**Comment by rcorral** [comment]
Yes, I've updated the description

**Comment by spahl** [comment]
Changelog should be a link to the actual diff between the two commits. Otherwise it's hard to evaluate.

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -514,7 +514,7 @@ package:
       "oauth":{"enabled":false},
 {% endswitch %}
 {% endswitch %}
-      "tracking":{"enabled":{{ telemetry_enabled }}}}}}
+      "tracking":{"enabled":{{ telemetry_enabled }}{{ ui_telemetry_metadata }}}}}}
```
Comment (alberts) [inline gen/dcos-config.yaml]: @rcorral just move the comma here?

**Pull request merged**